### ISSUE TITLE
Make work with fzf-lua

### ISSUE THREAD

**Issue by NullSense** [ojroques/nvim-lspfuzzy#39]
I will probably pick this up when I have a bit of time. I have migrated to fzf-lua and would like to be able to use that for previews.

**Comment by SunJun8** [comment]
> I will probably pick this up when I have a bit of time. I have migrated to fzf-lua and would like to be able to use that for previews.

Yes, I also migrated to fzf lua, but I have to say, this plugin is more friendly to the nvim native lsp api.

**Comment by eekryuos** [comment]
Sorry. I would like to know why closing this issue? The lspfuzzy's main branch didn't work with fzf-lua until now.
